architecture/core_concepts/routes.adoc

```diff
@@ -621,6 +629,30 @@ the destination. This field is required, but only for re-encryption.
 [[router-sharding]]
 == Router Sharding
 
+Sharding allows the operator to define multiple router groups. Each
+router in the group serves only a subset of traffic (sharding).
+
+When routers are sharded, a given route is bound to a router in the group. The route
+binding ensures uniqueness of the route across the shard.  Uniqueness allows secure
+and non-secure versions of the same route to exist within a single shard.
+This implies that routes now have a visible life cycle that moves from created to
+bound to active.
+
+In the sharded environment the first route to hit the shard reserves the right to
+exist there indefinitely. Even across restarts.
+
+During a green/blue deployment a route may exist multiple times.  An OpenShift
```

Comment:
@knobunc @ramr, This is the architecture, core concepts, routes section. If sharding doesn't go here, where does it go? I am leaving this here until we discuss it a bit more.

OK I am confused, I thought sharding permitted a selection of external names for a service to go one router and other external names to go to a different router. The external DNS points to the IP of the selected router (that handles the desired subset of names). I didn't relate this to anything like a a/b bleedover.